### PR TITLE
Alpha homora strat

### LINKED ISSUE #6: AH lending strategy
Enable AH lending:
Interfaces and mock: shttps://github.com/AlphaFinanceLab/alpha-lending-smart-contract/tree/master/contracts
- [ ] deposit
- [ ] withdraw
- [ ] value
- [ ] claim & swap rewards on harvest
- [ ] unit tests
- [ ] deploy

### PR DESCRIPTION
**Pull request by 0xKirkjufell**
If you made a contract change, make sure to complete the checklist below before merging it in master.

Contract change checklist:
  - [x] Code reviewed by 1 reviewers.
  - [x] Unit tests pass – `yarn test`
  - [x] Smoke tests pass: `./scripts/smokeTest.sh`
  - [x] Slither tests pass with no warning: `yarn slither` (repair with `yarn slither:triage`)
  - [x] Echidna tests pass if PR includes changes to XUSD contract (not automated, run manually on local): `yarn echidna`

closes #6

### CODE REVIEW

File: deployments/network.fuji.json [hunk 1]
```diff
@@ -1,5457 +0,0 @@
-{
```
Comment (0xKirkjufell) [inline deployments/network.fuji.json:1]: maybe we don't want to do this

File: deploy/000_mock.js [hunk 2]
```diff
@@ -74,6 +74,42 @@ const deployMocks = async ({ getNamedAccounts, deployments }) => {
...
+    contract: "MockSafeBox",
+    from: deployerAddr,
+  });
+  // Deploy SafeBoxes
+  await deploy("MockSafeBoxUSDC.e", {
+    args: [dCUSDC.address, "Interest Bearing USDC.e v2", "ibUSDC.ev2"],
+    contract: "MockSafeBox",
+    from: deployerAddr,
+  });
+  // TODO Deploy mock IncentivesController
```
Comment (0xKirkjufell) [inline deploy/000_mock.js:111]: this is done, remove comment

File: deploy/001_oracles.js [hunk 3]
```diff
@@ -65,6 +65,11 @@ const deployOracles = async () => {
         .connect(sDeployer)
         .setFeed(assetAddresses.WAVAX, oracleAddresses.chainlink.AVAX_USD)
     );
+    withConfirmation(
```
Comment (0xKirkjufell) [inline deploy/001_oracles.js:68]: TODO: deploy a new oracleRouter on mainnet.

File: contracts/strategies/AlphaHomoraStrategy.sol [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+     */
+    function collectRewardTokens() external override onlyVault nonReentrant {
+        for (uint256 i = 0; i < rewardTokenAddresses.length; i++) {
+            IAlphaIncentiveDistributor _incentivesController = IAlphaIncentiveDistributor(
+                    incentiveControllerAddresses[i]
+                );
+            require(_incentivesController.token() == rewardTokenAddresses[i]);
+            uint256 _amount = _amounts[rewardTokenAddresses[i]];
+            bytes32[] memory _proof = _proofs[rewardTokenAddresses[i]];
+            uint256 _claimed = _incentivesController.claimed(address(this));
```
Comment (0xKirkjufell) [inline contracts/strategies/AlphaHomoraStrategy.sol:61]: will _claimed be 0 for first reward period, or does this throw if this is not in claimed?

File: contracts/strategies/AlphaHomoraStrategy.sol [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+        returns (uint256 balance)
+    {
+        uint256 safeBoxBalance = IERC20(_safeBox).balanceOf(address(this));
+        uint256 exchangeRate = _cToken.exchangeRateStored();
+        // e.g. 50e8*205316390724364402565641705 / 1e18 = 1.0265..e18
+        balance = (safeBoxBalance * exchangeRate) / 1e18;
+    }
+
+    /**
+     * @dev Retuns bool indicating whether asset is supported by strategy
```
Comment (0xKirkjufell) [inline contracts/strategies/AlphaHomoraStrategy.sol:209]: typo

**Review by 0xAnnapurna — APPROVED** [review]
LGTM!

**Pull request merged**